wazo_call_logd/database/alembic/versions/d6a8d09c2f29_add_index_call_logd_call_log_.py

```diff
@@ -0,0 +1,40 @@
+"""add_index_call_logd_call_log_participant__idx__call_log_id
+
+Revision ID: d6a8d09c2f29
+Revises: 4dfbea039971
+
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = 'd6a8d09c2f29'
+down_revision = '4dfbea039971'
+INDEX_NAME = 'call_logd_call_log_participant__idx__call_log_id'
+INDEX_TO_DROP = 'call_logd_call_log_participant__idx__user_uuid'
```

Comment:
Why do you remove the user_uuid index. How does that make the call_log query any faster?

Reply:
I was having great delays on the query (list_from_user) ending up in `TimeOutError: Connection closed by Client`, so I did a little investigation and found that when having multiple indexes on the same table; the performance of the operations (insert/update/delete) can be altered (I have put one link in the last commit description). I confirmed my suspicion and the link statement by removing the `user_uuid` index and it turned out to really improve the performance when we list from a user, whilst when I keep it; the delays are great and the `TimeoutError` becomes frequent the more we increase the number of calls per user (the performance drops after going over  > 4000~5000 calls per user)

Reply:
To be honest, I'm afraid that removing this index could have an unexpected side effect. I'd be more confortable leaving it as it can be expensive (very long) to add it back once it's been removed.